src/CallableResolver.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Middlewares\Utils;
+
+use RuntimeException;
+
+/**
+ * Simple interface to resolve callableish values.
+ */
+interface CallableResolver
```

Comment:
In my opinion, this interface should be moved into the CallableResolver namespace.